lib/runtime/extrinsic/unchecked_extrinsic_test.go

```diff
@@ -0,0 +1,94 @@
+// Copyright 2020 ChainSafe Systems (ON) Corp.
+// This file is part of gossamer.
+//
+// The gossamer library is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Lesser General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// The gossamer library is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU Lesser General Public License for more details.
+//
+// You should have received a copy of the GNU Lesser General Public License
+// along with the gossamer library. If not, see <http://www.gnu.org/licenses/>.
+package extrinsic
+
+import (
+	"fmt"
+	"log"
+	"math/big"
+	"os"
+	"testing"
+
+	"github.com/ChainSafe/gossamer/lib/common"
+	"github.com/ChainSafe/gossamer/lib/keystore"
+	"github.com/stretchr/testify/require"
+)
+
+var kr keystore.Sr25519Keyring
```

Comment:
i think this needs to be instantiated, `var kr, _ = keystore.NewSr25519Keyring()`

Reply:
oh nvm I see you instantiated it in the test main

Reply:
I'm initializing this in TestMain, but I can move var declaration or comment for more clarity.

Reply:
I'm fine with either!